src/Sylius/Bundle/ChannelBundle/spec/Context/FakeChannel/FakeChannelPersisterSpec.php

```diff
@@ -75,12 +75,7 @@ function it_persists_fake_channel_codes_in_a_cookie(
                 if ($cookie->getName() !== '_channel_code') {
                     return false;
                 }
-
-                if ($cookie->getValue() !== 'fake_channel_code') {
-                    return false;
-                }
-
-                return true;
+                return $cookie->getValue() === 'fake_channel_code';
```

Comment:
`return $cookie->getName() === '_channel_code' && $cookie->getValue() == 'fake_channel_code';` seems like a better solution here.

Reply:
That's inconsistent with single-compare-at-line approach in previous code.

Also less readable, since I have to keep 2 condigions and 3 compare signs in my mind while reading 1 line.

Reply:
I could agree with that, but I still don't think this `if` changed into a `return` is a better solution, especially when the possible future diffs are taken into account (same as in https://github.com/Sylius/Sylius/pull/9902/files#r231438895).

Reply:
I see and agree